src/flows/components/AddButtons.tsx

```diff
@@ -80,15 +82,27 @@ const AddButtons: FC<Props> = ({index, onInsert}) => {
     return acc
   }, {})
 
+  const pipeIds = flow.data.allIDs
+  const firstQueryIndex = pipeIds.findIndex(
+    id =>
+      flow.data.byID[id].type === 'queryBuilder' ||
+      flow.data.byID[id].type === 'rawFluxEditor'
+  )
+
   const cellFamilies = SUPPORTED_FAMILIES.filter(fam =>
     pipeFamilies.hasOwnProperty(fam.family)
   ).map(fam => {
     const pipes = pipeFamilies[fam.family].map(def => {
+      let status = ComponentStatus.Default
+      if (index < firstQueryIndex && NEED_QUERY_PANELS.includes(def.type)) {
```

Comment:
a note to future me and not to ari: i dont like how this is keying off of the type and not something more generic like the family, or a property of the panels themselves, but that's the specific ask of the product, so deal